app/interactors/add_movie_to_round.rb

```diff
@@ -3,26 +3,42 @@
 class AddMovieToRound
   include Hanami::Interactor
 
-  def initialize(import_service: ImportTmdbMovie.new)
-    @import_service = import_service
+  expose :movie, :round
+
+  def initialize(import_tmdb_movie_interactor: ImportTmdbMovie.new)
```

Comment:
I have nothing against the "service" nomenclature, but since these are a libraries' interactors, and are called that way in the project tree, I'd rather be consistent with that. It's a rather long name, but it's correct.